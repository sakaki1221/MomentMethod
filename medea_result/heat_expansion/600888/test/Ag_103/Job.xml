<?xml version='1.0'?>
<!DOCTYPE MDRESULTS PUBLIC "-//MD//DTD MDRESULTS 1.0" "http://materialsdesign.com/DTD/mdresults_1_0.dtd" []>

<MDRESULTS>
  <System id="initial">
    <Date>16 Jan 2017 05:56:03 GMT</Date>
  <Characterization><PropertyValue property="symmetry" type="text" units="">P1</PropertyValue><PropertyValue property="setting" type="text" units="">P1</PropertyValue><PropertyValue property="empirical formula" type="text" units="">Ag</PropertyValue><PropertyValue property="primitive cell formula" type="text" units="">Ag32</PropertyValue><PropertyValue property="primitive Z" type="text" units="">32</PropertyValue><PropertyValue property="symmetry" type="text" units="">P1</PropertyValue><PropertyValue property="setting" type="text" units="">P1</PropertyValue><PropertyValue property="empirical formula" type="text" units="">Ag</PropertyValue><PropertyValue property="primitive cell formula" type="text" units="">Ag32</PropertyValue><PropertyValue property="primitive Z" type="text" units="">32</PropertyValue></Characterization></System>
  <Calculation id="initial">
    <Date>16 Jan 2017 05:56:03 GMT</Date>
    <System>initial</System>
  <Results><PropertyValue property="VASP energy" type="float" units="eV">-89.346621</PropertyValue><PropertyValue property="Fermi energy" type="float" units="eV">2.366800</PropertyValue><PropertyValue dimensions="3 32" property="atomic derivatives" type="float" units="kJ/mol/m">0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000</PropertyValue><PropertyValue property="cell volume" type="float" units="Ang^3">631.050</PropertyValue><PropertyValue property="pressure" type="float" units="GPa">-6.154</PropertyValue><PropertyValue dimensions="6" property="stress" type="float" units="GPa">6.15400 6.15400 6.15400 0.00000 0.00000 0.00000</PropertyValue><PropertyValue property="energy of formation" type="float" units="kJ/mol">3.4312</PropertyValue><PropertyValue property="heat of formation" type="float" units="kJ/mol">-69.6527</PropertyValue><PropertyValue property="VASP energy" type="float" units="eV">-89.346621</PropertyValue><PropertyValue property="Fermi energy" type="float" units="eV">2.366800</PropertyValue><PropertyValue dimensions="3 32" property="atomic derivatives" type="float" units="kJ/mol/m">0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000</PropertyValue><PropertyValue property="cell volume" type="float" units="Ang^3">631.050</PropertyValue><PropertyValue property="pressure" type="float" units="GPa">-6.154</PropertyValue><PropertyValue dimensions="6" property="stress" type="float" units="GPa">6.15400 6.15400 6.15400 0.00000 0.00000 0.00000</PropertyValue><PropertyValue property="energy of formation" type="float" units="kJ/mol">3.4312</PropertyValue><PropertyValue property="heat of formation" type="float" units="kJ/mol">-69.6527</PropertyValue></Results></Calculation>
<System id="displaced_Ag__a"><Date>18 Jan 2017 04:45:53 GMT</Date><Characterization><PropertyValue property="symmetry" type="text" units="">P1</PropertyValue><PropertyValue property="setting" type="text" units="">P1</PropertyValue><PropertyValue property="empirical formula" type="text" units="">Ag</PropertyValue><PropertyValue property="primitive cell formula" type="text" units="">Ag32</PropertyValue><PropertyValue property="primitive Z" type="text" units="">32</PropertyValue></Characterization></System><Calculation id="displaced_Ag__a">
    <Date>18 Jan 2017 04:45:53 GMT</Date>
    <System>displaced_Ag__a</System>
    <Results>
      <PropertyValue property="VASP energy" type="float" units="eV">-89.346091</PropertyValue>
      <PropertyValue property="Fermi energy" type="float" units="eV">2.366700</PropertyValue>
      <PropertyValue dimensions="3 32" property="atomic derivatives" type="float" units="kJ/mol/m">52981986.369 10236434.770 0.000
687640927.510 806316092.649 0.000
687640957.078 14.784 -806316092.649
52981986.744 -10236434.770 0.000
-145222387.780 2137277.629 -2137277.589
-69405277.506 0.000 0.000
53431939.927 -10573899.652 0.000
687640927.510 -14.784 806316092.649
687640957.078 -806316092.649 0.000
52981986.744 0.188 -10236434.770
-145222387.859 2137277.550 2137277.589
-5947818554.523 0.000 0.000
-145222387.702 -2137277.550 -2137277.589
-13386107.007 0.000 0.000
745234965.072 -881345784.864 0.000
745234932.752 -16.160 881345784.864
53431939.539 10573899.652 0.000
133073652.009 -674929.777 674929.765
-110238528.292 0.000 0.000
52981986.369 -0.188 10236434.770
-145222387.780 -2137277.629 2137277.589
-69405277.506 0.000 0.000
745234965.072 16.160 -881345784.864
745234932.752 881345784.864 0.000
53431939.539 -0.194 10573899.652
133073652.034 -674929.753 -674929.765
244999504.714 0.000 0.000
133073651.985 674929.753 674929.765
-32846581.899 0.000 0.000
53431939.927 0.194 -10573899.652
133073652.009 674929.777 -674929.765
-110238528.292 0.000 0.000</PropertyValue>
      <PropertyValue property="cell volume" type="float" units="Ang^3">631.050</PropertyValue>
      <PropertyValue property="pressure" type="float" units="GPa">-6.115</PropertyValue>
      <PropertyValue dimensions="6" property="stress" type="float" units="GPa">6.11300 6.11500 6.11500 0.00000 0.00000 0.00000</PropertyValue>
      <PropertyValue property="energy of formation" type="float" units="kJ/mol">3.4328</PropertyValue>
      <PropertyValue property="heat of formation" type="float" units="kJ/mol">-69.1880</PropertyValue>
    </Results>
  </Calculation><System id="displaced_Ag__b"><Date>18 Jan 2017 04:45:54 GMT</Date><Characterization><PropertyValue property="symmetry" type="text" units="">P1</PropertyValue><PropertyValue property="setting" type="text" units="">P1</PropertyValue><PropertyValue property="empirical formula" type="text" units="">Ag</PropertyValue><PropertyValue property="primitive cell formula" type="text" units="">Ag32</PropertyValue><PropertyValue property="primitive Z" type="text" units="">32</PropertyValue></Characterization></System><Calculation id="displaced_Ag__b">
    <Date>18 Jan 2017 04:45:54 GMT</Date>
    <System>displaced_Ag__b</System>
    <Results>
      <PropertyValue property="VASP energy" type="float" units="eV">-89.346091</PropertyValue>
      <PropertyValue property="Fermi energy" type="float" units="eV">2.366700</PropertyValue>
      <PropertyValue dimensions="3 32" property="atomic derivatives" type="float" units="kJ/mol/m">-53431939.539 -10573899.652 0.000
-745122444.447 -881908226.335 0.000
-745122476.788 -16.170 881908226.335
-53431939.927 10573899.652 0.000
146234782.428 2249765.925 -2249765.884
69517765.800 0.000 0.000
-53094475.041 10348923.064 0.000
-745122444.447 16.170 -881908226.335
-745122476.788 881908226.335 0.000
-53431939.927 -0.194 10573899.652
146234782.345 2249765.842 2249765.884
5945231323.757 0.000 0.000
146234782.510 -2249765.842 -2249765.884
13611083.595 0.000 0.000
-687640957.093 807103510.708 0.000
-687640927.495 14.799 -807103510.708
-53094474.661 -10348923.064 0.000
-133186140.304 -674929.777 674929.765
110238528.292 0.000 0.000
-53431939.539 0.194 -10573899.652
146234782.428 -2249765.925 2249765.884
69517765.800 0.000 0.000
-687640957.093 -14.799 807103510.708
-687640927.495 -807103510.708 0.000
-53094474.661 0.190 -10348923.064
-133186140.279 -674929.753 -674929.765
-246236875.950 0.000 0.000
-133186140.328 674929.753 674929.765
33409023.370 0.000 0.000
-53094475.041 -0.190 10348923.064
-133186140.304 674929.777 -674929.765
110238528.292 0.000 0.000</PropertyValue>
      <PropertyValue property="cell volume" type="float" units="Ang^3">631.050</PropertyValue>
      <PropertyValue property="pressure" type="float" units="GPa">-6.115</PropertyValue>
      <PropertyValue dimensions="6" property="stress" type="float" units="GPa">6.11400 6.11500 6.11500 0.00000 0.00000 0.00000</PropertyValue>
      <PropertyValue property="energy of formation" type="float" units="kJ/mol">3.4328</PropertyValue>
      <PropertyValue property="heat of formation" type="float" units="kJ/mol">-69.1880</PropertyValue>
      <PropertyValue property="zero-point energy" type="float" units="kJ/mol">1.4125</PropertyValue>
      <PropertyValue property="E(0)" type="float" units="kJ/mol">4.8437</PropertyValue>
      <PropertyValue property="E(PV)" type="float" units="kJ/mol">-73.0839</PropertyValue>
      <PropertyValue property="Eelec" type="float" units="kJ/mol">3.4312</PropertyValue>
      <PropertyValue property="Temperatures" type="float" units="K">0.001 1 2 3 4 5 10 15 20 30 40 50 60 70 80 90 100 125 150 175 200 225 250 273 275 298 300 350 400 450 500 550 600 650 700 750 800 850 900 950 1e+003 1.05e+003 1.1e+003 1.15e+003 1.2e+003 1.25e+003 1.3e+003 1.35e+003 1.4e+003 1.45e+003 1.5e+003 1.55e+003 1.6e+003 1.65e+003 1.7e+003 1.75e+003 1.8e+003 1.85e+003 1.9e+003 1.95e+003 2e+003 2.1e+003 2.2e+003 2.3e+003 2.4e+003 2.5e+003 2.6e+003 2.7e+003 2.8e+003 2.9e+003 3e+003</PropertyValue>
      <PropertyValue property="S(T)" type="float" units="J/K/mol">0.0000 0.0000 0.0007 0.0036 0.0089 0.0169 0.1272 0.5158 1.3272 3.9726 7.2526 10.6082 13.8084 16.7813 19.5191 22.0374 24.3584 29.4413 33.7205 37.4014 40.6247 43.4886 46.0634 48.2220 48.4013 50.3769 50.5416 54.3445 57.6478 60.5669 63.1815 65.5489 67.7116 69.7023 71.5461 73.2632 74.8699 76.3795 77.8030 79.1498 80.4277 81.6433 82.8025 83.9103 84.9710 85.9884 86.9660 87.9068 88.8134 89.6881 90.5333 91.3507 92.1423 92.9095 93.6538 94.3766 95.0790 95.7622 96.4271 97.0748 97.7062 98.9228 100.0829 101.1914 102.2528 103.2708 104.2488 105.1901 106.0971 106.9723 107.8178</PropertyValue>
      <PropertyValue property="Cv(T)" type="float" units="kJ/mol">0.0000 0.0001 0.0033 0.0122 0.0265 0.0474 0.4187 1.7825 4.0751 9.2766 13.5084 16.4934 18.5470 19.9771 20.9967 21.7420 22.3001 23.2001 23.7124 24.0297 24.2392 24.3844 24.4891 24.5615 24.5670 24.6222 24.6265 24.7099 24.7643 24.8016 24.8284 24.8483 24.8634 24.8752 24.8845 24.8921 24.8982 24.9034 24.9077 24.9113 24.9144 24.9171 24.9194 24.9214 24.9232 24.9247 24.9261 24.9274 24.9285 24.9295 24.9304 24.9312 24.9319 24.9326 24.9332 24.9337 24.9343 24.9347 24.9352 24.9356 24.9360 24.9366 24.9372 24.9377 24.9381 24.9385 24.9389 24.9392 24.9395 24.9397 24.9400</PropertyValue>
      <PropertyValue property="H(T)-Eelec" type="float" units="kJ/mol">-71.6715 -71.6715 -71.6715 -71.6714 -71.6714 -71.6714 -71.6705 -71.6655 -71.6511 -71.5841 -71.4691 -71.3181 -71.1423 -70.9493 -70.7441 -70.5302 -70.3099 -69.7400 -69.1531 -68.5560 -67.9525 -67.3446 -66.7336 -66.1695 -66.1203 -65.5547 -65.5054 -64.2718 -63.0349 -61.7957 -60.5549 -59.3130 -58.0702 -56.8267 -55.5827 -54.3383 -53.0936 -51.8485 -50.6033 -49.3578 -48.1122 -46.8664 -45.6205 -44.3745 -43.1283 -41.8822 -40.6359 -39.3896 -38.1432 -36.8967 -35.6502 -34.4037 -33.1571 -31.9105 -30.6639 -29.4172 -28.1705 -26.9238 -25.6771 -24.4303 -23.1835 -20.6899 -18.1963 -15.7025 -13.2087 -10.7149 -8.2211 -5.7272 -3.2333 -0.7393 1.7547</PropertyValue>
      <PropertyValue property="E(T)-Eelec" type="float" units="">1.4125 1.4125 1.4125 1.4125 1.4125 1.4125 1.4134 1.4184 1.4328 1.4998 1.6149 1.7658 1.9416 2.1347 2.3398 2.5537 2.7740 3.3439 3.9309 4.5279 5.1315 5.7394 6.3503 6.9145 6.9636 7.5293 7.5785 8.8121 10.0490 11.2882 12.5290 13.7709 15.0137 16.2572 17.5012 18.7456 19.9904 21.2354 22.4807 23.7261 24.9718 26.2176 27.4635 28.7095 29.9556 31.2018 32.4480 33.6944 34.9408 36.1872 37.4337 38.6802 39.9268 41.1734 42.4200 43.6667 44.9134 46.1601 47.4069 48.6536 49.9004 52.3940 54.8877 57.3814 59.8752 62.3690 64.8629 67.3568 69.8507 72.3446 74.8386</PropertyValue>
    </Results>
  </Calculation></MDRESULTS>
