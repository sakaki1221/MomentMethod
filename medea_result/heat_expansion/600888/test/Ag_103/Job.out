Startup is 'sourced'
Opening the database
Phonon calculation for Untitled

Phonon parameters
=================
The calculation explores the full Brillouin zone and accounts for an interaction range of about 10.0 Angstroem.
The asymmetric atoms are displaced by +/-0.02 Angstroem.

In a first step, need to run an optimization of atom positions for the supercell containing 32 atoms.
Then, need to run a total number of 2 single point energy calculations on supercells with displaced atoms containing 32 atoms.

      Supercell transformation matrix              Supercell lattice vectors
       2.00000    0.00000    0.00000             8.57736    0.00000    0.00000
       0.00000    2.00000    0.00000             0.00000    8.57736    0.00000
       0.00000    0.00000    2.00000             0.00000    0.00000    8.57736

     Supercell           Site        Displaced atom     Displacement (fractional units)
   ---------------    ----------     --------------     -------------------------------
  undisplaced           ----            ----             0.00000    0.00000    0.00000
  displaced_Ag__a       Ag              Ag               0.00233    0.00000    0.00000
  displaced_Ag__b       Ag              Ag              -0.00233    0.00000    0.00000

Invariance conditions are not enforced.
No approximation to LO/TO splitting
Properties: Phonon dispersion and form factors 
            Phonon density of states 
            Thermodynamic functions
==========================================


VASP parameters
===============
Since no magnetic moments are in the model, this is a non-magnetic calculation using 'accurate' precision
and a user-defined planewave cutoff energy of 600.000 eV.

The electronic iterations convergence is 1.00E-005 eV using the Very Fast (RMM-DIIS) algorithm
and reciprocal space projection operators.

The requested k-spacing is 0.1 per Angstrom, which leads to a 26x26x26 mesh.
This corresponds to actual k-spacings of 0.098 x 0.098 x 0.098 per Angstrom.

Using first order Methfessel-Paxton smearing with a width of 0.2 eV.

Other non-default parameters:
	Extrafine augmentation grid for accurate forces is TRUE
	Algorithm is Very Fast (RMM-DIIS)
	Wave functions is TRUE
==========================================

------------------------------------------------------------------------

Results for initial structural minimization of the supercell without displacements:
--------------------------------------------------------------------------------------

There are 20 symmetry-unique k-points
The plane wave cutoff is 600.00 eV

   VASP energy:             -89.346621 eV for Ag32

   Initial VASP energy:     -89.346621 eV for Ag32
   Relaxation energy:         0.000000 eV gained after 1 optimization steps.

Electronic contributions:
                          Empirical Formula          Cell
                                    Ag          (Ag)32
                          --------------- ---------------
              VASP Energy        -269.395       -8620.636 kJ/mol
      Energy of formation           3.431         109.799
                  PV term         -73.084       -2338.686

   This is the electronic part of the formation energy with respect to the
   elements in their standard state. It is obtained from the total energy (above)
   and tabulated VASP energies for the elements in their standard state.
   See the documentation for more details.

   Details for Ag:
                                             VASP energy (Etot):     -269.39 kJ/mol
       Standard state energy of 1.00 atoms of Ag  1.00 * -272.83:  -  -272.83
                                                                   ----------
                                        Energy of formation (Ef):        3.43 kJ/mol


Cell parameters:
 Parameter    Value    
---------- ------------
         a     8.577364
         b     8.577364
         c     8.577364
     alpha    89.999999
      beta    89.999999
     gamma    89.999999

    Volume    78.880853 Ang^3

        Density:     9.083 Mg/m^3

   The pressure given below is exerted by the system according to its volume.
   Positive pressure would cause expansion during full geometry optimization.
       Pressure:    -6.154 GPa
               =   -61.540 kbar

                      XX        YY        ZZ        XY        YZ        ZX
         Stress:     6.154     6.154     6.154     0.000     0.000     0.000 GPa
               =    61.540    61.540    61.540     0.000     0.000     0.000 kbar
   The stress tensor above is imposed on the system, i.e. negative values of diagonal components 
   would cause expansion of the corresponding lattice parameter upon full geometry optimization.
   The pressure and stress include only electronic terms, i.e. the vibrational,
   temperature and other terms are not included here.

Fractional Coordinates:
	Atom          Initial Coordinates             Final Coordinates
	-----      -------- -------- --------      -------- -------- --------
	Ag           0.7500   0.7500   0.5000        0.7500   0.7500   0.5000
	Ag           0.7500   0.7500   0.0000        0.7500   0.7500   0.0000
	Ag           0.7500   0.0000   0.2500        0.7500   0.0000   0.2500
	Ag           0.7500   0.2500   0.5000        0.7500   0.2500   0.5000
	Ag           0.0000   0.7500   0.2500        0.0000   0.7500   0.2500
	Ag           0.0000   0.0000   0.5000        0.0000   0.0000   0.5000
	Ag           0.2500   0.7500   0.5000        0.2500   0.7500   0.5000
	Ag           0.7500   0.0000   0.7500        0.7500   0.0000   0.7500
	Ag           0.7500   0.2500   0.0000        0.7500   0.2500   0.0000
	Ag           0.7500   0.5000   0.2500        0.7500   0.5000   0.2500
	Ag           0.0000   0.7500   0.7500        0.0000   0.7500   0.7500
	Ag           0.0000   0.0000   0.0000        0.0000   0.0000   0.0000
	Ag           0.0000   0.2500   0.2500        0.0000   0.2500   0.2500
	Ag           0.0000   0.5000   0.5000        0.0000   0.5000   0.5000
	Ag           0.2500   0.7500   0.0000        0.2500   0.7500   0.0000
	Ag           0.2500   0.0000   0.2500        0.2500   0.0000   0.2500
	Ag           0.2500   0.2500   0.5000        0.2500   0.2500   0.5000
	Ag           0.5000   0.7500   0.2500        0.5000   0.7500   0.2500
	Ag           0.5000   0.0000   0.5000        0.5000   0.0000   0.5000
	Ag           0.7500   0.5000   0.7500        0.7500   0.5000   0.7500
	Ag           0.0000   0.2500   0.7500        0.0000   0.2500   0.7500
	Ag           0.0000   0.5000   0.0000        0.0000   0.5000   0.0000
	Ag           0.2500   0.0000   0.7500        0.2500   0.0000   0.7500
	Ag           0.2500   0.2500   0.0000        0.2500   0.2500   0.0000
	Ag           0.2500   0.5000   0.2500        0.2500   0.5000   0.2500
	Ag           0.5000   0.7500   0.7500        0.5000   0.7500   0.7500
	Ag           0.5000   0.0000   0.0000        0.5000   0.0000   0.0000
	Ag           0.5000   0.2500   0.2500        0.5000   0.2500   0.2500
	Ag           0.5000   0.5000   0.5000        0.5000   0.5000   0.5000
	Ag           0.2500   0.5000   0.7500        0.2500   0.5000   0.7500
	Ag           0.5000   0.2500   0.7500        0.5000   0.2500   0.7500
	Ag           0.5000   0.5000   0.0000        0.5000   0.5000   0.0000

Results for supercell calculations:
------------------------------------------------------------------------
	        Supercell         Energy (eV)
	      -------------      -------------
	        undisplaced       -89.34662
	     displaced_Ag__a      -89.34609
	     displaced_Ag__b      -89.34609


Symmetry analysis of Gamma point phonon frequencies:
----------------------------------------------------
   Crystal  Point  Group: O_h   (m3m) 
   Supercell Point Group: O_h   (m3m) 
    Table of irreducible representations of point group:
 ------------------------------------------
    IR     1  5  7  9 11  6  8 10 12 13 16 17 18 21 23 14 15 19 20 22 24  2  3  4 25 38 39 43 44 46 48 29 30 31 32 33 34 35 36 26 27 28 37 40 41 42 45 47
 ------------------------------------------
 A1g(R)    1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1   x^2+y^2+z^2                         
 A2g       1  1  1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1  1  1  1  1 -1 -1 -1 -1 -1 -1  1  1  1  1  1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1                                       
 Eg(R)     2 -1 -1 -1 -1 -1 -1 -1 -1  0  0  0  0  0  0  0  0  0  0  0  0  2  2  2  2  0  0  0  0  0  0 -1 -1 -1 -1 -1 -1 -1 -1  2  2  2  0  0  0  0  0  0   (2z^2-x^2-y^2, x^2-y^2)             
 T1g       3  0  0  0  0  0  0  0  0 -1 -1 -1 -1 -1 -1  1  1  1  1  1  1 -1 -1 -1  3  1  1  1  1  1  1  0  0  0  0  0  0  0  0 -1 -1 -1 -1 -1 -1 -1 -1 -1                                       
 T2g(R)    3  0  0  0  0  0  0  0  0  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1  3 -1 -1 -1 -1 -1 -1  0  0  0  0  0  0  0  0 -1 -1 -1  1  1  1  1  1  1   (xy,xz,yz)                          
 A1u       1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1                                       
 A2u       1  1  1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1  1  1  1 -1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1  1  1  1  1  1  1                                       
 Eu        2 -1 -1 -1 -1 -1 -1 -1 -1  0  0  0  0  0  0  0  0  0  0  0  0  2  2  2 -2  0  0  0  0  0  0  1  1  1  1  1  1  1  1 -2 -2 -2  0  0  0  0  0  0                                       
 T1u(I)    3  0  0  0  0  0  0  0  0 -1 -1 -1 -1 -1 -1  1  1  1  1  1  1 -1 -1 -1 -3 -1 -1 -1 -1 -1 -1  0  0  0  0  0  0  0  0  1  1  1  1  1  1  1  1  1   (x,y,z)                             
 T2u       3  0  0  0  0  0  0  0  0  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -3  1  1  1  1  1  1  0  0  0  0  0  0  0  0  1  1  1 -1 -1 -1 -1 -1 -1                                       
 ------------------------------------------
    Wave vector= 0.0000  0.0000  0.0000    no macro. electric field
 Multi. Omega   Irreducible Representations
  3     0.013     T1u(I)                                                                                                     
Notation: (R)  Raman active mode
          (I)  Infrared active mode
          (RI) Raman and Infrared active mode

Thermodynamic Properties
   for empirical formula Ag

The following thermodynamic functions are obtained from the phonon
density of states within the harmonic approximation. The results are
meaningful only below the melting point, which lies well below 3000 K
for most materials.
Definitions:
      Cv            : vibrational heat capacity at constant volume
E_vib(T)=E(T)-E(0)  : the change in vibrational internal energy from 0 K, where
                        E(0) = E_elec + ZPE is the electronic energy of formation E_elec plus the zero point energy ZPE
      S(T)          : the vibrational entropy at temperature T
   -(A(T)-E(0))     : the change in the vibrational Helmholtz free energy from 0 K
      E(T)          : the electronic energy of formation plus the vibrational internal energy, so this term is
                        E_elec + ZPE + E_vib(T)
      A(T)          : the electronic plus vibrational Helmholtz free energy,
                        E(T) - T.S(T)
Note that the electronic energy, which is part of the enthalpy and free energies,
 is referenced to the elements in their standard state.

Electronic energy =     3.43 kJ/mol (referenced to elements in the standard state)
          PV term =   -73.08 kJ/mol
Zero-point energy =     1.41 kJ/mol

    T       Cv       E(T)-E(0)      S(T)   -(A(T)-E(0))     E(T)        A(T)
    K     J/K/mol      kJ/mol     J/K/mol     kJ/mol       kJ/mol      kJ/mol
  ----- ----------- ----------- ----------- ----------- ----------- -----------
     0      0.0000      0.0000      0.0000      0.0000      4.8437      4.8437
     1      0.0001      0.0000      0.0000      0.0000      4.8437      4.8437
     2      0.0033      0.0000      0.0007      0.0000      4.8437      4.8437
     3      0.0122      0.0000      0.0036      0.0000      4.8437      4.8437
     4      0.0265      0.0000      0.0089      0.0000      4.8437      4.8437
     5      0.0474      0.0001      0.0169      0.0000      4.8438      4.8437
    10      0.4187      0.0010      0.1272      0.0003      4.8446      4.8434
    15      1.7825      0.0060      0.5158      0.0018      4.8497      4.8419
    20      4.0751      0.0204      1.3272      0.0062      4.8640      4.8375
    30      9.2766      0.0874      3.9726      0.0318      4.9311      4.8119
    40     13.5084      0.2024      7.2526      0.0877      5.0461      4.7560
    50     16.4934      0.3533     10.6082      0.1771      5.1970      4.6666
    60     18.5470      0.5292     13.8084      0.2993      5.3729      4.5444
    70     19.9771      0.7222     16.7813      0.4525      5.5659      4.3912
    80     20.9967      0.9273     19.5191      0.6342      5.7710      4.2095
    90     21.7420      1.1412     22.0374      0.8421      5.9849      4.0016
   100     22.3001      1.3616     24.3584      1.0743      6.2053      3.7694
   125     23.2001      1.9314     29.4413      1.7487      6.7751      3.0950
   150     23.7124      2.5184     33.7205      2.5396      7.3621      2.3040
   175     24.0297      3.1155     37.4014      3.4297      7.9592      1.4139
   200     24.2392      3.7190     40.6247      4.4059      8.5627      0.4378
   225     24.3844      4.3269     43.4886      5.4580      9.1706     -0.6143
   250     24.4891      4.9379     46.0634      6.5779      9.7816     -1.7342
   273     24.5615      5.5020     48.2220      7.6625     10.3457     -2.8188
   275     24.5670      5.5511     48.4013      7.7592     10.3948     -2.9155
   298     24.6222      6.1168     50.3769      8.8954     10.9605     -4.0517
   300     24.6265      6.1661     50.5416      8.9963     11.0097     -4.1526
   350     24.7099      7.3996     54.3445     11.6208     12.2433     -6.7771
   400     24.7643      8.6365     57.6478     14.4224     13.4802     -9.5787
   450     24.8016      9.8757     60.5669     17.3792     14.7194    -12.5355
   500     24.8284     11.1165     63.1815     20.4740     15.9602    -15.6303
   550     24.8483     12.3585     65.5489     23.6932     17.2022    -18.8495
   600     24.8634     13.6013     67.7116     27.0255     18.4450    -22.1818
   650     24.8752     14.8447     69.7023     30.4615     19.6884    -25.6178
   700     24.8845     16.0887     71.5461     33.9932     20.9324    -29.1495
   750     24.8921     17.3331     73.2632     37.6139     22.1768    -32.7702
   800     24.8982     18.5779     74.8699     41.3176     23.4216    -36.4739
   850     24.9034     19.8229     76.3795     45.0992     24.6666    -40.2555
   900     24.9077     21.0682     77.8030     48.9541     25.9119    -44.1104
   950     24.9113     22.3137     79.1498     52.8782     27.1574    -48.0345
  1000     24.9144     23.5593     80.4277     56.8679     28.4030    -52.0242
  1050     24.9171     24.8051     81.6433     60.9199     29.6488    -56.0762
  1100     24.9194     26.0510     82.8025     65.0312     30.8947    -60.1875
  1150     24.9214     27.2970     83.9103     69.1992     32.1407    -64.3555
  1200     24.9232     28.5431     84.9710     73.4214     33.3868    -68.5777
  1250     24.9247     29.7893     85.9884     77.6956     34.6330    -72.8519
  1300     24.9261     31.0356     86.9660     82.0196     35.8793    -77.1759
  1350     24.9274     32.2819     87.9068     86.3915     37.1256    -81.5478
  1400     24.9285     33.5283     88.8134     90.8096     38.3720    -85.9659
  1450     24.9295     34.7747     89.6881     95.2722     39.6184    -90.4285
  1500     24.9304     36.0212     90.5333     99.7779     40.8649    -94.9342
  1550     24.9312     37.2677     91.3507    104.3251     42.1114    -99.4814
  1600     24.9319     38.5143     92.1423    108.9124     43.3580   -104.0688
  1650     24.9326     39.7609     92.9095    113.5388     44.6046   -108.6951
  1700     24.9332     41.0076     93.6538    118.2029     45.8513   -113.3592
  1750     24.9337     42.2542     94.3766    122.9038     47.0979   -118.0601
  1800     24.9343     43.5009     95.0790    127.6402     48.3446   -122.7966
  1850     24.9347     44.7476     95.7622    132.4113     49.5913   -127.5677
  1900     24.9352     45.9944     96.4271    137.2160     50.8381   -132.3723
  1950     24.9356     47.2411     97.0748    142.0537     52.0848   -137.2100
  2000     24.9360     48.4879     97.7062    146.9232     53.3316   -142.0795
  2100     24.9366     50.9815     98.9228    156.7551     55.8252   -151.9114
  2200     24.9372     53.4752    100.0829    166.7058     58.3189   -161.8621
  2300     24.9377     55.9689    101.1914    176.7698     60.8126   -171.9261
  2400     24.9381     58.4627    102.2528    186.9423     63.3064   -182.0986
  2500     24.9385     60.9565    103.2708    197.2188     65.8002   -192.3751
  2600     24.9389     63.4504    104.2488    207.5950     68.2941   -202.7513
  2700     24.9392     65.9443    105.1901    218.0672     70.7880   -213.2236
  2800     24.9395     68.4382    106.0971    228.6318     73.2819   -223.7881
  2900     24.9397     70.9322    106.9723    239.2854     75.7758   -234.4417
  3000     24.9400     73.4261    107.8178    250.0251     78.2698   -245.1814

Job completed on Wed 18 January 2017 at 13:45:57 æ±äº¬ (æ¨™æº–æ™‚) after 5166 s (1:26:06)
